src/hooks/useKeyboardNavigation.js

```diff
@@ -27,11 +29,71 @@ const useKeyboardNavigation = ({
 
   useEffect(() => {
     return tinykeys(emojiListRef.current, {
-      ArrowRight: focusNextEmoji,
-      ArrowLeft: focusPrevEmoji,
+      ArrowRight: moveRight,
+      ArrowLeft: moveLeft,
+      ArrowUp: moveUp,
+      ArrowDown: moveDown,
     });
   }, []);
 
+  const moveRight = () => {
+    navigateGrid(RIGHT);
+  };
+  const moveLeft = () => {
+    navigateGrid(LEFT);
+  };
+  const moveUp = () => {
+    navigateGrid(UP);
+  };
+  const moveDown = () => {
+    navigateGrid(DOWN);
+  };
```

Comment:
As I mentioned in my edge-case observations, these functions miss some edge (literally) behaviors. See there how I think they should behave there.

If you want to merge this and handle these issues in a follow up PR that's fine as well.